packages/smooth_app/lib/pages/scan/continuous_scan_page.dart

```diff
@@ -1,114 +1,140 @@
 import 'package:flutter/material.dart';
-import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:flutter_svg/flutter_svg.dart';
 import 'package:provider/provider.dart';
 import 'package:qr_code_scanner/qr_code_scanner.dart';
 import 'package:smooth_app/data_models/continuous_scan_model.dart';
 import 'package:smooth_app/pages/personalized_ranking_page.dart';
 import 'package:smooth_app/widgets/smooth_product_carousel.dart';
 import 'package:smooth_ui_library/smooth_ui_library.dart';
+import 'package:smooth_ui_library/util/ui_helpers.dart';
 
 class ContinuousScanPage extends StatelessWidget {
   final GlobalKey _scannerViewKey = GlobalKey(debugLabel: 'Barcode Scanner');
 
   @override
   Widget build(BuildContext context) {
-    final ContinuousScanModel model = context.watch<ContinuousScanModel>();
-    final AppLocalizations localizations = AppLocalizations.of(context)!;
-    final Size screenSize = MediaQuery.of(context).size;
-    final double carouselHeight = screenSize.height / 2;
-    final double viewFinderBottomOffset = carouselHeight / 2.0;
-    return Scaffold(
-      appBar: AppBar(toolbarHeight: 0.0),
-      body: Stack(
-        children: <Widget>[
-          Container(
-            alignment: Alignment.center,
-            color: Colors.black,
-            child: Padding(
-              // Double the offset to account for the vertical centering.
-              padding: EdgeInsets.only(bottom: 2 * viewFinderBottomOffset),
-              child: SvgPicture.asset(
-                'assets/actions/scanner_alt_2.svg',
-                width: 60.0,
-                height: 60.0,
-                color: Colors.white,
+    return LayoutBuilder(
+        builder: (BuildContext context, BoxConstraints constraints) {
+      final ContinuousScanModel model = context.watch<ContinuousScanModel>();
+      final Size screenSize = MediaQuery.of(context).size;
+      final Size scannerSize = Size(
+        screenSize.width * 0.6,
+        screenSize.width * 0.33,
+      );
```

Comment:
Is it right that both use the width

Reply:
yeah, they are supposed to be proportional to one another